packages/spor-i18n-react/src/index.tsx

```diff
@@ -8,3 +8,9 @@ export enum Language {
 export const { LanguageProvider, useTranslation } = initLobot<typeof Language>(
   Language.NorwegianBokmal
 );
+
+export type Translations = {
+  [key: string]: Translations | { [key in Language]: Translation };
+};
+
+type Translation = string | ((args: unknown) => string);
```

Comment:
Oh yeah, this is totally unrelated, but I came up with this type while writing the docs. 

Basically, you can add a type declaration to your texts like so:

```tsx
const texts: Translations = {
  myText: { nb: 'Norsk', en: 'English', sv: 'Svenska' }
}
```
, and have that `texts` object be type safe.